sdk/keyvault/keyvault-common/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
nit: this should extend the common one rather than creating more debt ;)

Reply:
Woah!! Since when??? ;)

Reply:
This should be good now!